src/Generator/CombGenerator.php

```diff
@@ -107,7 +107,7 @@ public function generate(int $length): string
 
         return (string) hex2bin(
             str_pad(
-                bin2hex((string) $hash),
```

Comment:
Redundant cast: it is impossible for `$hash` to be anything other than a string at this location due to initialization to a string and string type hints on invoked methods.